src/platform/graphics/constants.js

```diff
@@ -976,20 +1065,32 @@ export const pixelFormatInfo = new Map([
     [PIXELFORMAT_BGRA8,         { name: 'BGRA8', size: 4 }],
 
     // compressed formats
-    [PIXELFORMAT_DXT1,              { name: 'DXT1', blockSize: 8, srgb: true }],
-    [PIXELFORMAT_DXT3,              { name: 'DXT3', blockSize: 16, srgb: true }],
-    [PIXELFORMAT_DXT5,              { name: 'DXT5', blockSize: 16, srgb: true }],
+    [PIXELFORMAT_DXT1,              { name: 'DXT1', blockSize: 8, srgb: true, linearFormat: PIXELFORMAT_DXT1_SRGB }],
```

Comment:
I'm still wondering why DXT1 format is flagged as `srgb: true` and why the mapping is called 'linearFormat'? This is the exact opposite of what it should be.

DXT1 format is 100% linear - it is stored and sampled in linear space (no conversion anywhere) and has nothing to do with srgb.

DXT1_SRGB is data stored in gamma space. The gpu will convert the texture data to linear space for sampling and interpolation and then return the result in linear space for us. This format should be flagged as `srgb: true`.

So this line of code should be something like:
```javascript
[PIXELFORMAT_DXT1, { name: 'DXT1', blockSize: 8, srgbFormat: PIXELFORMAT_DXT1_SRGB }],
```

Also not sure what `srgb` flag is currently used for, but you could probably just use the presence of `srgbFormat` and remove the flag entirely.